### PR TITLE
Use esbuild

### PR DESCRIPTION
**Pull request by worranhin**
I try using the Esbuild and output the compressed file to "./out/main.js".

The file after compressing is about 60kb. I don't know if it fits your need. @dzylikecode 

#5

### CODE REVIEW

**Comment by dzylikecode** [comment]
Yes, but don't commit the compressed file

**Review by dzylikecode — CHANGES_REQUESTED** [review]
remove it, please!

File: package.json [hunk 1]
```diff
@@ -69,14 +69,20 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "node ./test/runTest.js"
+    "test": "node ./test/runTest.js",
```
Comment (dzylikecode) [inline package.json:72]: Can you explain how it works?

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
 {
   "name": "md-paste-enhanced",
-  "version": "0.0.2",
```
Comment (dzylikecode) [inline package-lock.json:3]: Does it change automatically?

File: out/main.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...&t.bigHour===void 0),Object.isFrozen==null||!Object.isFrozen(e))e._isValid=r;else return r}return e._isValid}function be(e){var t=V(NaN);return e!=null?te(f(t),e):f(t).userInvalidated=!0,t}var Et=o.momentProperties=[],at=!1;function it(e,t){var s,r,n,a=Et.length;if(R(t._isAMomentObject)||(e._isAMomentObject=t._isAMomentObject),R(t._i)||(e._i=t._i),R(t._f)||(e._f=t._f),R(t._l)||(e._l=t._l),R(t._strict)||(e._strict=t._strict),R(t._tzm)||(e._tzm=t._tzm),R(t._isUTC)||(e._isUTC=t._isUTC),R(t._offset)||(e._offset=t._offset),R(t._pf)||(e._pf=f(t)),R(t._locale)||(e._locale=t._locale),a>0)for(s=0;s<a;s++)r=Et[s],n=t[r],R(n)||(e[r]=n);return e}function pe(e){it(this,e),this._d=new Date(e._d!=null?e._d.getTime():NaN),this.isValid()||(this._d=new Date(NaN)),at===!1&&(at=!0,o.updateOffset(this),at=!1)}function E(e){return e instanceof pe||e!=null&&e._isAMomentObject!=null}function Ht(e){o.suppressDeprecationWarnings===!1&&typeof console<"u"&&console.warn&&console.warn("Deprecation warning: "+e)}function C(e,t){var s=!0;return te(function(){if(o.deprecationHandler!=null&&o.deprecationHandler(null,e),s){var r=[],n,a,i,h=arguments.length;for(a=0;a<h;a++){if(n="",typeof arguments[a]=="object"){n+=`
```
Comment (dzylikecode) [inline out/main.js:1]: I think you'd better remove this file because it's unnecessary

**Comment by dzylikecode** [comment]
By the way, you can review the `ReadMe.md` and improve it if you are available

**Review by worranhin — COMMENTED** [review]
out directory deleted

File: out/main.js (hunk 3 above)
Comment (worranhin) [reply out/main.js]: As you wish.

**Comment by dzylikecode** [comment]
Cool, thanks @worranhin

File: package.json (hunk 1 above)
Comment (worranhin) [reply package.json:72]: I'm not sure what you mean. The scripts can be run by `npm run`. Then esbuild do the compressing. I didn't config anything.

**Pull request merged**